docs/devel/developer-guides/vagrant.md

```diff
@@ -130,225 +150,234 @@ To view the services on any of the nodes:
 
 You may need to build the binaries first, you can do this with `make`
 
-```console
+```shell
 $ ./cluster/kubectl.sh get nodes
-
-NAME                     LABELS                                          STATUS
-kubernetes-node-0whl     kubernetes.io/hostname=kubernetes-node-0whl     Ready
-kubernetes-node-4jdf     kubernetes.io/hostname=kubernetes-node-4jdf     Ready
-kubernetes-node-epbe     kubernetes.io/hostname=kubernetes-node-epbe     Ready
-```
-
-### Interacting with your Kubernetes cluster with the `kube-*` scripts.
-
-Alternatively to using the vagrant commands, you can also use the `cluster/kube-*.sh` scripts to interact with the vagrant based provider just like any other hosting platform for kubernetes.
-
-All of these commands assume you have set `KUBERNETES_PROVIDER` appropriately:
-
-```sh
-export KUBERNETES_PROVIDER=vagrant
 ```
 
-Bring up a vagrant cluster
-
-```sh
-./cluster/kube-up.sh
-```
-
-Destroy the vagrant cluster
-
-```sh
-./cluster/kube-down.sh
-```
-
-Update the vagrant cluster after you make changes (only works when building your own releases locally):
-
-```sh
-./cluster/kube-push.sh
-```
+### Authenticating with your master
 
-Interact with the cluster
+When using the vagrant provider in Kubernetes, the `cluster/kubectl.sh` script
+will cache your credentials in a `~/.kubernetes_vagrant_auth` file so you will
+not be prompted for them in the future.
 
-```sh
-./cluster/kubectl.sh
+```shell
+cat ~/.kubernetes_vagrant_auth
 ```
 
-### Authenticating with your master
-
-When using the vagrant provider in Kubernetes, the `cluster/kubectl.sh` script will cache your credentials in a `~/.kubernetes_vagrant_auth` file so you will not be prompted for them in the future.
-
-```console
-$ cat ~/.kubernetes_vagrant_auth
+```json
 { "User": "vagrant",
-  "Password": "vagrant"
+  "Password": "vagrant",
   "CAFile": "/home/k8s_user/.kubernetes.vagrant.ca.crt",
   "CertFile": "/home/k8s_user/.kubecfg.vagrant.crt",
   "KeyFile": "/home/k8s_user/.kubecfg.vagrant.key"
 }
 ```
 
-You should now be set to use the `cluster/kubectl.sh` script. For example try to list the nodes that you have started with:
+You should now be set to use the `cluster/kubectl.sh` script. For example try to
+list the nodes that you have started with:
 
-```sh
+```shell
 ./cluster/kubectl.sh get nodes
 ```
 
 ### Running containers
 
-Your cluster is running, you can list the nodes in your cluster:
-
-```console
-$ ./cluster/kubectl.sh get nodes
-
-NAME                     LABELS                                          STATUS
-kubernetes-node-0whl     kubernetes.io/hostname=kubernetes-node-0whl     Ready
-kubernetes-node-4jdf     kubernetes.io/hostname=kubernetes-node-4jdf     Ready
-kubernetes-node-epbe     kubernetes.io/hostname=kubernetes-node-epbe     Ready
-```
-
-Now start running some containers!
+You can use `cluster/kube-*.sh` commands to interact with your VM machines.
+Before starting a container there will be no pods, services, or Deployments:
```

Comment:
this is not true.  there are a number of resources in the `kube-system` namespace.